test/test_prototype_transforms.py

```diff
@@ -1488,16 +1488,10 @@ def test__transform(self, mocker):
 
         fn.assert_has_calls(expected_calls)
 
-        expected_within_targets = sum(is_within_crop_area)
-
         # check number of bboxes vs number of labels:
         output_bboxes = output[1]
         assert isinstance(output_bboxes, datapoints.BoundingBox)
-        assert len(output_bboxes) == expected_within_targets
-
-        output_masks = output[2]
-        assert isinstance(output_masks, datapoints.Mask)
-        assert len(output_masks) == expected_within_targets
+        assert (output_bboxes[~is_within_crop_area] == 0).all()
```

Comment:
We no longer remove boxes so we need to check that the degenerate ones are marked correctly